library/hawkular_alerts_group_member.py

```diff
@@ -162,13 +179,14 @@ def main():
             hawkular_api_auth_token=dict(
                 default=os.environ.get('HAWKULAR_TOKEN'), type='str'),
             tenant=dict(required=True, type='str'),
-            state=dict(required=True, type='str', choices=['present', 'absent']),
+            state=dict(required=True, type='str', choices=['present', 'list']),
```

Comment:
It wasn't an option before since the client itself doesn't support member deletion.
I plan to add it to the client and only after to this module